src/pages/auth/Login.js

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from "react";
+import { Link,useNavigate} from "react-router-dom";
+import { AuthContainer } from "./component/AuthContainer";
+import { SigninImage } from "../../assets/images";
+import {
+  BsFillEyeFill,
+  FaEnvelope,
+  AiOutlineArrowRight,
+} from "../../assets/icons";
+import "./auth.css"
+
+
+const Login = () => {
+  const [email, setEmail] = useState();
+
+  const [password, setPassword] = useState();
```

Comment:
Suggestion: You can create on state variable called formdetails to manage these

Reply:
Yes I will be doing that only that is why I have mentioned don't give reviews on functionality because I will be adding it later.